package.json

```diff
@@ -20,11 +20,11 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "debug": "^2.1.2",
-    "tap": "^1.3.2"
+    "debug": "^3.2.5",
+    "tap": "^10.7"
   },
   "dependencies": {
-    "nan": "^2.0.5"
+    "nan": "^2.13.2"
```

Comment:
This shouldn't be necessary, the previous dep spec already stated "latest 2.x". If you found that nan < 2.13.2 is installed in your local tree, you just need to reinstall your deps (and possibly are using some sort of package locking scheme).

Reply:
Indeed.

This PR was initially motivated by the `npm audit` error messages after install this module (due to old devDep versions).

Reading nodejs/nan#849 caused me to hope a solution might be updating nan. Alas that isn't the case (as you point out, I had the latest version). I still get a bunch of compiler errors and a build failure under node 12. I'm over my head and don't know the solution.

Reply:
I've run into this before, see bnoordhuis/node-iconv#183 and bnoordhuis/node-iconv@5516f62cc9.

If another package depends on an old version of nan but one that's still compatible with `^2.0.5`, then npm's dedupe logic will reuse that version for our package because semver. Annoying but technically correct.

Reply:
Folks getting an older version of `nan` has also shown up in this repo in #13 and #15

Reply:
In that case, I agree the nan depspec should get updated to one supporting 12.x, though it does still leave me confused, because that other package that is depending on the older but great than 2.0.5 nan, it won't work on 12.x either, will it?

Reply:
Not yet. But it may for a future version of `nan`. In any case, we know the version of `nan` listed as a dep is inadequate, so why not bump it?

Reply:
I think depending on npm to do it right is the right thing. In this case, since the nan 2.0 doesn't support a feature we want (12.x support), I agree that updating the depspec makes sense. Unfortunately, I spent 10 minutes going through their issues and changelog, and couldn't find what version added 12.x support, but since the latest does, updating this to the latest is reasonable.